assets/src/scripts/components/dailyValueHydrograph/selectors/time-series-data.js

```diff
@@ -40,6 +43,16 @@ export const getCurrentTimeSeriesPoints = createSelector(
                     'grades'
                 ], zippedStep);
             });
+
+        return result.sort((first, second) => {
```

Comment:
I had assumed that the time series data comes in ordered. It doesn't which is why this step is needed.

Reply:
I wonder if this could be addressed in the service in the future.... :thinking:

Reply:
I did ask that question and I was told that for now this is how it will be. I am wondering if I should sort the data before putting it in the Redux store but opted for doing it in the selector. I like the idea of just sticking raw data in the Redux store and having the selectors do the work.

Reply:
yeah, downstream developers are going to be confused without a sort.  Of course, a sort is also really expensive and could potentially slow down a query.  Everything is always tradeoffs.  We might do something like what WQP does, where we allow for sorting as a parameter, but if there is sorting the size of the query is limited.